src/freenet/node/Node.java

```diff
@@ -1603,6 +1603,12 @@ public Long get() {
 			@Override
 			public void set(Long amountOfDataToCheckCompressionRatio) {
 				synchronized(Node.this) {
+					if (amountOfDataToCheckCompressionRatio < 0 || amountOfDataToCheckCompressionRatio > 100 * 1024 * 1024) {
+						Logger.normal(Node.class, "Amount of data to check for compression should be 100 MiB max"
+								+ amountOfDataToCheckCompressionRatio);
```

Comment:
This will make the log output look like "Amount of data to check for compression should be 100 MiB max12312412414" which looks like the node just threw up in the logfile. 😄

Reply:
Fixed in 8c222a28c13e17437fb78cb98356f58372140042